addons/intel/e825.go

```diff
@@ -207,9 +207,6 @@ func (d *E825PluginData) populateDpllPins() error {
 	return nil
 }
 
-// Setup mockable pin setting function
-var e825DoPinSet = BatchPinSet
```

Comment:
This is removed because we now have a global function pointer shared with the e810 and common mocking functions instead.